src/lib_json/json_value.cpp

```diff
@@ -1398,11 +1398,11 @@ String Value::Comments::get(CommentPlacement slot) const {
 }
 
 void Value::Comments::set(CommentPlacement slot, String comment) {
+  if (slot >= CommentPlacement::numberOfCommentPlacement) {
```

Comment:
This is not the code I suggested. Can you do it again?

Reply:
Yes. If return directly while `slot >= CommentPlacement::numberOfCommentPlacement`, the **CI** told that Some checks were not successful.

Reply:
I'd be interested in the CI failure.